### PR TITLE
Provide uninstall instructions in the README

### LINKED ISSUE #746: Documentation needed for uninstalling rbenv
In the main readme, there is no section for how to uninstall rbenv. This may make users nervous to try rbenv without an easy way to remove it.

### PR DESCRIPTION
**Pull request by mislav**
It might be useful to demonstrate that rbenv doesn't keep your system hostage, and that it's easy to disable or uninstall.

Is this too wordy?

Fixes #746

/cc @jchristi @sstephenson

### CODE REVIEW

**Comment by sstephenson** [comment]
Looks great.

**Comment by jchristi** [comment]
Wow, amazing turnaround time! :+1:
